migrations/versions/20181224_user_measure_table_rename_user_page_table_to_user_measure.py

```diff
@@ -0,0 +1,113 @@
+"""Rename `user_page` table to `user_measure`
+
+Revision ID: 20181224_user_measure_table
+Revises: 20181220_populate_measures
+Create Date: 2018-12-24 09:52:07.668219
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "20181224_user_measure_table"
+down_revision = "20181220_populate_measures"
+branch_labels = None
+depends_on = None
+
+max_measure_version_temp_table = """
+CREATE TEMPORARY TABLE max_measure_version
+AS
+SELECT
+    mv.*
+FROM
+    measure_version AS mv
+WHERE
+    mv.page_type = 'measure'
+    AND mv.version = (
+        SELECT
+            MAX(version)
+        FROM
+            measure_version AS max_mv
+        WHERE
+            max_mv.guid = mv.guid);
+"""
+
+update_user_measure_table_with_measure_ids = """
+UPDATE
+    user_measure AS user_measure
```

Comment:
Does this need `AS user_measure` here? The reverse `update_user_page_table_with_page_ids` query doesn't have `AS user_page`.

Reply:
You're right, it doesn't.